### ISSUE TITLE
Using staged & unstaged flags don't function as expected

### ISSUE THREAD

**Issue by jaydorsey** [prontolabs/pronto#318]
Using `pronto run` works for me every time when I'm on a branch.

Using `pronto run --staged` and `pronto run --staged` never work as expected. Example of the behavior I'm seeing below. The rb file in the picture has an indentation error I introduced.

![image](https://user-images.githubusercontent.com/191564/47526795-d832a500-d86e-11e8-9022-3dd0592e13cf.png)

This creates an issue for my workflow because I'm using overcommit with a pronto pre-commit hook, so it's using the `--staged` flag by default (which I would expect). I've found a workaround with overcommit but I'm not sure this is desired behavior.

I tried 0.9.6/master but it doesn't seem to resolve the problem either.

**Comment by doomspork** [comment]
@jaydorsey this looks like a bug, would you be open to submitting a PR to resolve this?

**Comment by jaydorsey** [comment]
I'd love to volunteer, but I haven't used pronto since I filed the PR. I switched to using overcommit exclusively around that time.

I did investigate this at the time I filed the issue but couldn't identify a root cause.

I did take a look at this with a quick example today and it looks like one of the latest releases fixed the issue as reported.

- `pronto run --staged` works when I stage a file with `git add`
- `pronto run --unstaged` works on unstaged files as expected

`pronto run` doesn't seem to do anything whether the file is staged, unstaged, or commited. It used to run when it was committed, I think, so that might still be broken? 

I'm going to close this issue because it appears resolved as reported. Feel free to open it if you think the error still exists.
